docs/.vitepress/theme/index.ts

```diff
@@ -0,0 +1,20 @@
+import DefaultTheme from 'vitepress/theme'
+import './style/index.css'
+import ArticleMetadata from "./components/ArticleMetadata.vue"
+import { inBrowser } from 'vitepress'
+import busuanzi from 'busuanzi.pure.js'
+import MyLayout from './components/MyLayout.vue'
+
+export default {
+  extends: DefaultTheme,
+  Layout: MyLayout,
+  enhanceApp({ app , router }) {
+    if (inBrowser) {
+      router.onAfterRouteChanged = () => {
+        busuanzi.fetch()
```

Comment:
在 `docs/.vitepress/theme/index.ts` 中，`router.onAfterRouteChanged` 回调直接调用了 `busuanzi.fetch()`。如果 `busuanzi` 库内部没有做防抖处理，可能会在路由快速切换时导致不必要的请求。建议添加一个简单的防抖逻辑。

```suggestion
import DefaultTheme from 'vitepress/theme'
import './style/index.css'
import ArticleMetadata from "./components/ArticleMetadata.vue"
import { inBrowser } from 'vitepress'
import busuanzi from 'busuanzi.pure.js'
import MyLayout from './components/MyLayout.vue'

let fetchTimeout: number | null = null;

export default {
  extends: DefaultTheme,
  Layout: MyLayout,
  enhanceApp({ app , router }) {
    if (inBrowser) {
      router.onAfterRouteChanged = () => {
        if (fetchTimeout) {
          clearTimeout(fetchTimeout);
        }
        fetchTimeout = window.setTimeout(() => {
          busuanzi.fetch();
          fetchTimeout = null;
        }, 100); // 100ms 防抖
      }
    }
    // 注册全局组件
    app.component('ArticleMetadata' , ArticleMetadata)
  }
}